app/_hub/reedelk/reedelk-transformer/index.md

```diff
@@ -0,0 +1,205 @@
+---
+name: Reedelk Transformer
+
+publisher: Reedelk
+
+categories:
+  - transformations
+
+type:
+  plugin        
+
+desc: Kong plugin to transform Reedelk requests and responses
+
+description: |
+  The Reedelk  transformer plugin allows transforming the upstream request body or
+  downstream response body by invoking a Reedelk REST flow before hitting the
+  upstream server, or before sending the downstream response back to the client.
+  The plugin allows applying upstream and downstream transformations together as well.
+
+support_url: https://github.com/reedelk/kong-plugin-reedelk-transformer/issues
+  # (Optional) A specific URL of your own for this extension.
+  # Defaults to the url setting in your publisher profile.
+
+source_url: https://github.com/reedelk/kong-plugin-reedelk-transformer
+
+license_type: Apache-2.0
+  # (Optional) For open source, use the abbreviations in parentheses at:
+  # https://opensource.org/licenses/alphabetical
+
+license_url: https://github.com/reedelk/kong-plugin-reedelk-transformer/blob/master/LICENSE.txt
+
+kong_version_compatibility:
+  community_edition:
+    compatible:
+      - 2.1.x
+      - 1.5.x
+  # enterprise_edition:
+  #  compatible:
+  #    - 2.1.x
+  #    - 1.5.x
+
+params:
+  name: reedelk-transformer
+  api_id: false
+  service_id: true
+  consumer_id: false
+  route_id: true
+  protocols: ["http", "https"]
+  dbless_compatible: yes
+  dbless_explanation:
+
+  config:
+    - name: upstream_transformer_url
+      required: no
+      default:
+      value_in_examples: http://myhost/upstream/transform
+      description:
+        The URL of the Reedelk REST flow endpoint to invoke for the Upstream
```

Comment:
We generally capitalize "Upstream" in the docs because it refers to an Upstream object in Kong. I don't think this is what's happening here?

Reply:
@reedelk can you please advise on this comment?

Reply:
@tlblessing yes please "Upstream" should be lowercased -> "upstream".

Reply:
Made lowercase.